src/client.js

```diff
@@ -47,7 +47,10 @@ class Client extends GeneratedApiClient {
 
     this.baseUrl = parsedConfig.client.orgUrl.replace(/\/$/, '');
     this.apiToken = parsedConfig.client.token;
-    this.http = new Http();
+    this.http = new Http({
+      cacheStore: clientConfig.cacheStore,
+      cacheMiddleware: clientConfig.cacheMiddleware
```

Comment:
We need to add content to the README that describes this new feature.  Make sure to note that the cacheMiddleware can be nulled in order to disable caching.